src/main/java/com/mgmtp/radio/service/user/UserServiceImpl.java

```diff
@@ -219,4 +220,10 @@ public boolean changePassword(String userId, String oldPassword, String newPassw
         }
         return false;
     }
+
+    @Override
+    public Flux<StationDTO> getAllStationOfSpecificUserById(String userId, StationPrivacy privacy) {
```

Comment:
please change name to **getStationsByUserIdAndPrivacy(userId, StationPrivacy.station_public)**